### DISCUSSION TITLE
Audiogridder Osx and windows

### DISCUSSION THREAD

**Discussion by DannyGouchi** [apohl79/audiogridder#1090] [Q&A]
Hi I'm running a Mac mini M1 as my main production computer and I want to install a dsp server on another computer that I have and don't use any longer.
The Question is can I install audigridder server on a windows computer and install the audiogridder plugin on my Mac mini or does both server and plugin have to be the same opperating system for this to work.
Thanks DG.

**Comment by apohl79** [comment]
You can mix different operating systems.

**Reply by DannyGouchi** [reply]
Thank you apohl79, I will try and install it as I have a old 19inch rack computer with a very good cpu xeon quad extreme 4ghz, its a mackintosh at the moment but runs better as windows so will install server as windows and use my m1 mini as my host.
